### ISSUE TITLE
[BUG]: provide a one shot command to compile & run libcu++s tests for development

### ISSUE THREAD

**Issue by gonzalobg** [NVIDIA/cccl#477]
### Is this a duplicate?

- [X] I confirmed there appear to be no [duplicate issues](https://github.com/NVIDIA/cccl/issues) for this bug and that I agree to the [Code of Conduct](CODE_OF_CONDUCT.md)

### Type of Bug

Something else

### Component

libcu++

### Describe the bug

Please provide a one shot command to compile & run libcu++ tests for development with the goal of simplifying contributions.

The shell script interface should:
- accept some way to select a subset of tests (e.g. a directory)
- accept some options that are relevant for testing (e.g. c++ version, sm_arch, build & serve docs locally)

The shell script should not have any dependencies beyond docker and bash. 
That is, the shell script should:
- build a container that contains everything required
- compile and run tests within the container

### How to Reproduce

.

### Expected behavior

.

**Comment by miscco** [comment]
I believe this is already done. 

The script in question is: `ci/test_libcudacxx.sh` It takes three arguments. `host_compiler`, `standard_version` and `cuda_architectures`

This script works fine outside of our devcontainers given that the ctk is in path.

For the second point, there are already the devcontainers that provide a fully configured development environment.

Using VS-code starting those is just a single click

**Comment by miscco** [comment]
I have changed the labels to documentation improvements.

**Comment by gonzalobg** [comment]
Would it be possible to provide a similar script that runs the whole thing inside the dev container directly ?

**Comment by jrhemstad** [comment]
Yes, this exists and I've been working on a contributor/devcontainer guide documentation. I'll have a PR up this week for both.

**Comment by jrhemstad** [comment]
Here's the first part of the docs on setting up the environment in a container: https://github.com/NVIDIA/cccl/pull/481

**Comment by alliepiper** [comment]
We have this today:

```bash
.devcontainer/launch.sh -d --cuda 13.0 --host gcc14 --gpus all -- \
  ci/util/build_and_test_targets.sh \
    --preset libcudacxx \
    --lit-tests "std/algorithms/alg.nonmodifying/alg.any_of/any_of.pass.cpp"
```

More info at: https://github.com/NVIDIA/cccl/blob/main/docs/cccl/development/build_and_bisect_tools.rst#build_and_test_targetssh
